### PR TITLE
feat(Farm): List WBNB-stkBNB LP

### PR DESCRIPTION
**Pull request by ChefMomota**
Farm info
LP symbol: stkBNB-WBNB
stableswap pair: 0x0b03e3d6ec0c5e5bbf993ded8d947c6fb6eec18d
LP address: 0x9976f5c8befdee650226d5571d5f5551e8722b75

Pool info
Token address: 0x4C882ec256823eE773B25b414d36F92ef58a7c0C
Pool contract address: 0x98ac153577d65f2eef2256f3aef8ba9d7e4b756b
Pool token rewards per block: 0.1186

### CODE REVIEW

**Review by Chef-Yogi — APPROVED** [review]

File: packages/farms/constants/56.ts [hunk 1]
```diff
@@ -50,6 +50,17 @@ const farms: SerializedFarmConfig[] = [
     quoteToken: bscTokens.wbnb,
   },
   //    * V3 by order of release (some may be out of PID order due to multiplier boost)
+  {
+    pid: 151,
+    lpSymbol: 'WBNB-stkBNB LP',
+    lpAddress: '0x9976f5c8BEfDee650226d5571d5F5551e8722b75',
+    token: bscTokens.wbnb,
+    quoteToken: bscTokens.stkbnb,
```
Comment (chef-jojo) [inline packages/farms/constants/56.ts:58]: for v2 farm, better keep wbnb in quote token to get the usd price?

File: packages/smart-router/evm/constants/stableSwap/56.ts [hunk 2]
```diff
@@ -60,7 +60,7 @@ export const pools: StableSwapPool[] = [
     quoteToken: bscTokens.stkbnb,
     stableSwapAddress: '0x0b03e3d6Ec0c5e5bBf993dED8D947C6fb6eEc18D',
     infoStableSwapAddress: '0x150c8AbEB487137acCC541925408e73b92F39A50',
-    stableLpFee: 0.0004,
+    stableLpFee: 0.0002,
```
Comment (chef-jojo) [inline packages/smart-router/evm/constants/stableSwap/56.ts:63]: fee `4000000` no?

Comment (Chef-Yogi) [reply packages/smart-router/evm/constants/stableSwap/56.ts:63]: Yeah, the fee is 4000000/10000000000
but the admin fee ratio is 5000000000/10000000000 = 0.5 
the rest of 0.5 is LPfee 
so it's 0.0002

Comment (chef-jojo) [reply packages/smart-router/evm/constants/stableSwap/56.ts:63]: Ah i see

**Review by chefjackson — APPROVED** [review]

**Pull request merged**